workflow/harvesting.md

```diff
@@ -1,17 +1,17 @@
-# Harvesting
+# Data Archiving (aka "Harvesting")
```

Comment:
Yep, you're right. "Data Archiving" is the name of the track and "Harvesting" is the name of the activity. I'll change it back.